api/language-extensions/language-server-extension-guide.md

```diff
@@ -379,13 +395,13 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
     diagnostics.push(diagnostic);
   }
 
-  // Send the computed diagnostics to VS Code.
+  // Send the computed diagnostics to VSCode.
```

Comment:
We like to use "VS Code" with a space, so it was updated in the extension guide. We should update the sample to use he same name.